packages/express-api-reference/playground/src/index.ts

```diff
@@ -0,0 +1,146 @@
+ *   get:
+ *     description: Get a mysterious string.
+ *     responses:
+ *       200:
+ *         description: Returns a mysterious string.
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ *               example: Hello World!
+ */
+app.get('/foobar', (req, res) => {
+  res.send('Hello World!')
+})
+
+/**
+ * @openapi
+ * /hello:
+ *   post:
+ *     description: Send a personalized greeting.
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               name:
+ *                 type: string
+ *                 example: John
+ *     responses:
+ *       200:
+ *         description: Returns a personalized greeting.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 message:
+ *                   type: string
+ *                   example: Hello, John!
+ */
+app.post('/hello', (req, res) => {
+  const { name } = req.body
+  res.json({ message: `Hello, ${name}!` })
+})
+
+/**
+ * @openapi
+ * /goodbye:
+ *   delete:
+ *     description: Delete a user session.
+ *     parameters:
+ *       - in: query
+ *         name: sessionId
+ *         required: true
+ *         schema:
+ *           type: string
+ *     responses:
+ *       200:
+ *         description: Confirms session deletion.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 status:
+ *                   type: string
+ *                   example: Session 1234567890 deleted.
+ */
+app.delete('/goodbye', (req, res) => {
+  const { sessionId } = req.query
+  res.json({ status: `Session ${sessionId} deleted.` })
+})
+
+/**
+ * @openapi
+ * /status:
+ *   get:
+ *     description: Get the server status.
+ *     responses:
+ *       200:
+ *         description: Returns the server status.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 status:
+ *                   type: string
+ *                   example: Server is running smoothly!
+ *                 uptime:
+ *                   type: number
+ *                   example: 123.456
+ */
+app.get('/status', (req, res) => {
+  res.json({ status: 'Server is running smoothly!', uptime: process.uptime() })
+})
+
+const ApiDefinition = swaggerJsdoc({
+  failOnErrors: true,
+  definition: {
+    openapi: '3.1.0',
+    info: {
+      title: 'Express Example',
+      description:
+        'The `@scalar/express-api-reference` middleware renders a beautiful API reference based on your OpenAPI specification.',
+      version: '1.0.0',
+    },
+  },
+  // Update the path to include the current file
+  apis: [new URL(import.meta.url).pathname],
```

Comment:
It's important we don't hardcode paths because they won't work in Docker and when served at subpaths